src/main/java/org/codehaus/mojo/license/AbstractDownloadLicensesMojo.java

```diff
@@ -242,6 +244,31 @@
     @Parameter( defaultValue = "${project}", readonly = true )
     private MavenProject project;
 
+    /**
+     * List of regexps/replacements applied to the license urls prior to download.
+     *
+     * <p>License urls that match a regular expression will be replace by the corresponding
+     * replacement. Replacement is performed with {@link java.util.regex.Matcher#replaceAll(String)
+     * java.util.regex.Matcher#replaceAll(String)} so you can take advantage of
+     * capturing groups to facilitate flexible transformations.
+     *
+     * <pre>
+     * {@code
+     *
+     * <licenseUrlReplacements>
+     *   <licenseUrlReplacement>
+     *   <regexp>\Qhttps://glassfish.java.net/public/CDDL+GPL_1_1.html\E</regexp>
+     *   <replacement>https://oss.oracle.com/licenses/CDDL+GPL-1.1</replacement>
```

Comment:
It would be nice to also have a capturing group example here. Something like 

```
<licenseUrlRegexpReplacement>
     <regexp>https://(.*)</regexp>
     <replacement>http://$1</replacement>
  </licenseUrlRegexpReplacement>
```